src/main/webapp/css/user-page.css

```diff
@@ -23,21 +23,33 @@
 
 /* Give each message a rounded border. */
 .message-div {
-  border: thin solid #71a9f7;
+  border: thin solid #cf309c;
   border-radius: 5px;
   margin-top: 20px;
+  background: #ffd4ea;
+}
+
+/* Make login/logout button font white */
+a {
```

Comment:
FYI: It is generally better to make more specific css rules at apply that to the elements.